files/default/sql/ddl/updates/2.5.0.sql

```diff
@@ -8,4 +8,6 @@ ALTER TABLE `hopsworks`.`serving` ADD COLUMN `model_name` varchar(255) COLLATE l
 SET SQL_SAFE_UPDATES = 0;
 UPDATE `hopsworks`.`serving`
 SET `model_name` = (SELECT REPLACE(SUBSTRING(SUBSTRING_INDEX(`model_path`, '/', 5), LENGTH(SUBSTRING_INDEX(`model_path`, '/', 4)) + 1), '/', ''));
-SET SQL_SAFE_UPDATES = 1;
\ No newline at end of file
+SET SQL_SAFE_UPDATES = 1;
+
+ALTER TABLE `hopsworks`.`executions` DROP FOREIGN KEY `FK_347_365`;
```

Comment:
This change also needs to be in initial tables

Reply:
fixed